test/pages.js

```diff
@@ -386,6 +386,42 @@ describe('Pages', function() {
     assert.strictEqual(page.path, `${homeId.replace(':en:published', '')}/another-parent/parent/sibling`);
   });
 
+  it('inferred page relationships are correct', async function() {
+    const req = apos.task.getReq();
+    const pages = await apos.page.find(req, {}).toArray();
+    for (const page of pages) {
+      if (page.level === 0) {
```

Comment:
You can't infer last target id and position on the home page because it never moves and has no parent or peer.